remoulade/api/main.py

```diff
@@ -54,11 +54,19 @@ def get_scheduled_jobs():
 def enqueue_message():
     payload = MessageSchema().load(request.json)
     actor = get_broker().get_actor(payload["actor_name"])
+    options = payload["options"]
```

Comment:
if options is not, you will do **None which will fail: `options = payload["options"] if payload["options"] is not None else {}`